using prod db
droid-v2-production-cluster.cluster-cy4dofwtnffp.ap-east-1.rds.amazonaws.com
2022-04-01 20:30:05 LORA-PC-2 root[3976426] WARNING celery role using -> slave
2022-04-01 20:30:08 LORA-PC-2 root[3976426] INFO AsyncORM: patching models
2022-04-01 : === OPTION MAKER CLASSIC CHECK NEW TICKER STARTED ===
The start date is set as: 2018-04-01
The end date is set as: 2022-04-01
2018-04-01
['HWDN.L']
Found 1 New Ticker
Insert Vol Period
Calculating Expiry Date
Calculating Expiry Date
Calculating Expiry Date
Calculating Expiry Date
Calculating Expiry Date
Calculating Expiry Date
Calculating Expiry Date
Empty DataFrame
Columns: [spot_date, classic_vol, ticker, spot_price, vol_period, time_to_exp, stop_loss, take_profit, expiry_date]
Index: []
Traceback (most recent call last):
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/pandas/core/arrays/datetimelike.py", line 932, in _cmp_method
    other = self._validate_comparison_value(other)
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/pandas/core/arrays/datetimelike.py", line 451, in _validate_comparison_value
    raise InvalidComparison(other)
pandas.core.arrays.datetimelike.InvalidComparison: 4

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/loratech/DROID_V2.1/manage.py", line 24, in <module>
    main()
  File "/home/loratech/DROID_V2.1/manage.py", line 20, in main
    execute_from_command_line(sys.argv)
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/home/loratech/DROID_V2.1/core/djangomodule/management/commands/backtest.py", line 35, in handle
    daily_classic()
  File "/home/loratech/DROID_V2.1/main_executive.py", line 149, in daily_classic
    option_maker_classic_check_new_ticker(ticker=ticker, currency_code=currency_code, time_to_exp=time_to_exp, mod=mod, option_maker=option_maker, null_filler=null_filler)
  File "/home/loratech/DROID_V2.1/main_executive.py", line 309, in option_maker_classic_check_new_ticker
    populate_bot_classic_backtest(start_date=start_date, end_date=end_date, ticker=new_ticker, currency_code=currency_code, time_to_exp=time_to_exp, mod=mod)
  File "/home/loratech/DROID_V2.1/bot/option_file_classic.py", line 90, in populate_bot_classic_backtest
    cond = main_pred["expiry_date"].apply(lambda x: x.weekday()) > 4
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/pandas/core/ops/common.py", line 65, in new_method
    return method(self, other)
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/pandas/core/arraylike.py", line 45, in __gt__
    return self._cmp_method(other, operator.gt)
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/pandas/core/series.py", line 4978, in _cmp_method
    res_values = ops.comparison_op(lvalues, rvalues, op)
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/pandas/core/ops/array_ops.py", line 229, in comparison_op
    res_values = op(lvalues, rvalues)
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/pandas/core/ops/common.py", line 65, in new_method
    return method(self, other)
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/pandas/core/arraylike.py", line 45, in __gt__
    return self._cmp_method(other, operator.gt)
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/pandas/core/arrays/datetimelike.py", line 934, in _cmp_method
    return invalid_comparison(self, other, op)
  File "/home/loratech/DROID_V2.1/env/lib/python3.8/site-packages/pandas/core/ops/invalid.py", line 34, in invalid_comparison
    raise TypeError(f"Invalid comparison between dtype={left.dtype} and {typ}")
TypeError: Invalid comparison between dtype=datetime64[ns] and int
make: *** [Makefile:13: classic] Error 1
